aws-auth-cognito/src/main/java/com/amplifyframework/statemachine/StateMachine.kt

```diff
@@ -183,6 +183,9 @@ internal open class StateMachine<StateType : State, EnvironmentType : Environmen
      */
     private fun process(event: StateMachineEvent) {
         val resolution = resolver.resolve(currentState, event)
+        println("StateMachine CurrentState: ${currentState.type}")
+        println("StateMachine Event: ${event.type}")
+        println("StateMachine ResolutionState: ${resolution.newState.type}")
```

Comment:
Do we want to leave this in production code? not behind some Log level?

Reply:
This PR will not be merged. It's only to debug CI build for resolution